### PR TITLE
added getSocial integration

### PR DESCRIPTION
**Pull request by Riscum**
added get social integration across the website

### CODE REVIEW

**Comment by luisiul359** [comment]
@Riscum please check your commit, you have just added empty lines

File: app/javascript/components/common/LayoutHeader.js [hunk 1]
```diff
@@ -32,6 +32,13 @@ class LayoutHeader extends PureComponent {
     componentDidMount() {
         this.updateMenuVisibility();
         this.targetElement = document.querySelector('.header__mobile-menu ul');
+        //getSocial 
+        const po = document.createElement("script");
```
Comment (luisiul359) [inline app/javascript/components/common/LayoutHeader.js:36]: Is this the best way to do it? I.e. couldn't you just add a `script` tag?

Comment (Riscum) [reply app/javascript/components/common/LayoutHeader.js:36]: not with react.
as far as I understand it we need to append it after the components is mounted. but let's try to get a second opinion.

Comment (luisiul359) [reply app/javascript/components/common/LayoutHeader.js:36]: I'm fine with that explanation.

**Review by luisiul359 — APPROVED** [review]

**Review by jagoncalves14 — CHANGES_REQUESTED** [review]
I disagree on integrating it this way. Just like João Romão mentioned on WhatsApp, this should be integrated through Google Tag Manager.
I checked and Nuno Cruz was the one who has access to GTM. We should ask him to integrate the getSocial script.

**Comment by nancruz** [comment]
@jagoncalves14 Am I 🤔 ? Did you mean Nuno Carneiro?

**Comment by jagoncalves14** [comment]
@nancruz , at least **I think** that you were the one who implemented the GTM script on `applications.html.erb file` 😛

**Comment by Riscum** [comment]
closed as we followed another path

**Pull request closed without merging**